docs/import_specifications.ADR.md

```diff
@@ -50,6 +55,14 @@ For Excel files, the first two rows may be hidden in any provided templates. Add
 Excel files may contain multiple data types, one per tab. Empty tabs will be ignored, and tabs
 that don't match the expected structure will be treated as an error.
 
+Parsing will be on a "best effort" basis given the underlying `pandas` technology. We have
+decided not to include type information per column in the spec, which means that the type
+will be determined by `pandas`. Pandas specifies a single type per column, which means that
+if a single `string` value is accidentally inserted into a column that is otherwise
+full of `float`s, `pandas` will coerce all the values in that column to `string`s. Under normal
+operations this should not be an issue, but could cause unexpected type returns if a user adds
+an incorrect value to a column
```

Comment:
Yes. The cell would expect a numeric value, but get a string. If the string can be properly coerced into a number, then it should be fine. Otherwise, there'll be an error.